config/locales/types.en.yml

```diff
@@ -3,6 +3,10 @@ en:
     authentication_modes:
       'null': 'No authentication required'
       oauth2: 'OAuth2'
+    modes:
```

Comment:
please use a more specific name like "query_parameter_modes"
because "modes" makes people say "modes of what?"